detox/test/e2e/38.canvas-screenshots.test.js

```diff
@@ -0,0 +1,20 @@
+const { expectElementSnapshotToMatch } = require('./utils/snapshot');
+
+describe(':android: Canvas screenshots', () => {
+  let fancyElement;
+
+  beforeEach(async () => {
+    await device.reloadReactNative();
+    await element(by.text('Canvas-Screenshots')).tap();
+    fancyElement = element(by.id('fancyElement'));
+  });
+
+  it('should take a screenshot of a vertically-clipped element', async () => {
+    await expectElementSnapshotToMatch(fancyElement, 'canvasScreenshot.vert');
+  });
+
+  it('should take a screenshot of a horizontally-clipped element', async () => {
+    await element(by.id('switchOrientation')).tap();
+    await expectElementSnapshotToMatch(fancyElement, 'canvasScreenshot.horiz', 0.995);
+  });
+});
```

Comment:
This is basically just a copy of "26.element-screenshots.test.js", just calling a different menu entry.